objectivec/GPBWellKnownTypes.m

```diff
@@ -115,3 +115,40 @@ - (void)setTimeIntervalSince1970:(NSTimeInterval)timeIntervalSince1970 {
 }
 
 @end
+
+NSString *const GPBTypeGoogleApisComPrefix = @"type.googleapis.com/";
+
+@implementation GPBAny (GBPWellKnownTypes)
+
+- (NSString*)getAnyTypeName {
```

Comment:
1) I still prefer to make it private. Is it ok?
2) If it has to be public, string filed return @"" when it's not set. Shall we make it consistent? (If consistent, there is no way to tell type_url is set to @"" or @"type.googleapis.com/")
3) If it's public, what happens if type_url is manually set to @"abcde"?